### PR TITLE
Solution for #6889

### PR DESCRIPTION
**Pull request by SpamReceiver**
See also: https://github.com/nextcloud/documentation/pull/7046

### CODE REVIEW

**Comment by szaimen** [comment]
@SpamReceiver sorry for coming back to you so late. Can you please rebase this to retrigger the CI? Thank you! :)

File: config/config.sample.php [hunk 1]
```diff
@@ -634,25 +634,40 @@
...
+ * _may be_ deleted. A file may be deleted after the minimum number of days 
+ * is expired if space is needed. The file will not be deleted if space is 
+ * not needed. 
+ *
+ * Whether "space is needed" depends on whether a user quota is defined or not:
+ *
+ *  * If no user quota is defined, the available space on the Nextcloud data 
+ *    partition sets the limit for the trashbin
+ *    (issues: see https://github.com/nextcloud/server/issues/28451 
+ *    and https://github.com/nextcloud/server/issues/28450).
```
Comment (PVince81) [inline config/config.sample.php:648]: please remove the issue mentions, I don't think they are useful as the code currently works as intended
enhancements can be considered separately

**Review by PVince81 — CHANGES_REQUESTED** [review]
Nice addition, the bullet points are what I've always missed when referring people to the docs, thanks for adding!

File: config/config.sample.php (hunk 1 above)
Comment (SpamReceiver) [reply config/config.sample.php]: I have removed the reference to #28450 which has been closed in the meantime.

I have not removed the reference to #28451 whichis still open and which is worth noting here. And I don't agree that the function works as intended. checking the available space on the root partition is simply wrong if the data is stored somewhere else.

**Comment by SpamReceiver** [comment]
Help! What do I need to do to complete this ticket?

**Comment by PVince81** [comment]
@SpamReceiver you need to sign your commits, see https://github.com/nextcloud/server/pull/29929/checks?check_run_id=6858670749

**Comment by SpamReceiver** [comment]
> @SpamReceiver you need to sign off your commits, see https://github.com/nextcloud/server/pull/29929/checks?check_run_id=6858670749

Thank you very much for this link. 

Unfortunately, I have done the edits via the Web GUI and have no local git. Is there a way to fix it?

**Comment by szaimen** [comment]
Hi @SpamReceiver, if you do not want to do it locally, you can otherwise simply create your change from the github web gui on a new branch and open a new PR with that! Do not forget to sign-off your changes directly though!

**Comment by SpamReceiver** [comment]
> Hi @SpamReceiver, if you do not want to do it locally, you can otherwise simply create your change from the github web gui on a new branch and open a new PR with that! Do not forget to sign-off your changes directly though!

Okay, I downloaded 2.5G via the clone command and followed the instructions here: https://github.com/nextcloud/server/pull/29929/checks?check_run_id=6858670749

Now, this thing still displays "Merging is blocked".

What am I supposed to do now?

**Comment by szaimen** [comment]
> What am I supposed to do now?

Try to do your changes again here https://github.com/nextcloud/server/edit/master/config/config.sample.php and create a new PR from that :)

**Comment by SpamReceiver** [comment]
> > What am I supposed to do now?
> 
> Try to do your changes again here https://github.com/nextcloud/server/edit/master/config/config.sample.php and create a new PR from that :)

New pull request: https://github.com/nextcloud/server/pull/33470

Please review!

**Comment by szaimen** [comment]
Superseded by https://github.com/nextcloud/server/pull/33470

**Pull request closed without merging**